files.sh

```diff
@@ -11,21 +11,27 @@ fn nash_complete_paths(parts, line, pos) {
 	if $status == "0" {
 		# already a directory
 		echo -n $lastpart | -grep "/$" >[1=]
-
+		
 		# complete with '/' if it wasnt given
 		if $status != "0" {
 			return ("/" "0")
 		}
-
-		dir     <= echo $lastpart | sed $sedArgs "s#/$##g"
+		
+		dir <= echo $lastpart | sed $sedArgs "s#/$##g"
+		
 		dirpath = $dir+"/"
-		fname  	= ""
+		fname   = ""
 	} else {
-		dir     <= dirname $lastpart | tr -d "\n"
-		dirpath = $dir+"/"
-		fname  	<= basename $lastpart | tr -d "\n"
+		dir <= dirname $lastpart | tr -d "\n"
+		
+		if $dir != "/" {
```

Comment:
Not that it would be better without this if, but it is not strictly necessary, at least not on linux. "/" and "//" will present the same result. But you are probably aiming at portability or something like that =)

Reply:
Hmm the fixes on layout got me confused, it seems that this is the fix, that is odd...is there a difference on behavior when you use //filepath and /filepath ?

While I was writing the question it struck me that this behavior is probably caused by the shell/bash/zsh/etc =P, makes sense to not behave this way on nash them, sorry for the noise =)